### PR TITLE
Add pre-generated galaxy_pulp bindings

### PR DESCRIPTION
**Pull request by cutwater**
This change adds pre-generated galaxy_pulp bindings to the
source control and removes building bindings at the
container image build step.

Issue: #52

### CODE REVIEW

File: .travis/install.sh [hunk 1]
```diff
@@ -63,7 +63,6 @@ images:
       pulpcore: pulpcore
       plugins:
         - ./galaxy_ng
-        - ./galaxy_ng/bindings/galaxy-pulp
```
Comment (fabricio-aguiar) [inline .travis/install.sh]: you have to remove it couple lines below also
https://github.com/ansible/galaxy_ng/blob/master/.travis/install.sh#L79

Comment (cutwater) [reply .travis/install.sh]: missed that. now it's done.

File: setup.py [hunk 2]
```diff
@@ -2,12 +2,23 @@
 
 from setuptools import find_packages, setup
 
-requirements = [
+# NOTE(cutwater): Because bindings are statically generated, requirements list
+#   from pulp-galaxy/setup.py has to be copied here and manually maintained.
+galaxy_pulp_requirements = [
```
Comment (alikins) [inline setup.py]: +1 for separating the reqs

File: setup.py [hunk 3]
```diff
@@ -2,12 +2,23 @@
 
 from setuptools import find_packages, setup
 
-requirements = [
+# NOTE(cutwater): Because bindings are statically generated, requirements list
+#   from pulp-galaxy/setup.py has to be copied here and manually maintained.
```
Comment (alikins) [inline setup.py]: Would it make any sense to add a setup.py to galaxy_pulp and install as it's own package?

(Probably not worth it considering end goal is to not have galaxy_pulp at all, so no need to change)

**Comment by alikins** [comment]
Also seems to work with some minimal testing. ie, `./compose build && ./compose up` and poking around the UI and api a little bit. +1

**Pull request merged**